### PR TITLE
View articles test

### PR DESCRIPTION
**Pull request by Carltesio**
# [PT Story](https://www.pivotaltracker.com/story/show/171811213)
## Changes proposed in this pull request:
* Adds cypress test for article visualization

## What I have learned working on this feature:
* Stubbing data through a cypress test

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,6 +3,11 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import axios from "axios";
+
+
```
Comment (hmvitous) [inline src/index.js]: Maybe delete extra lines that are empty.

**Review by hmvitous — COMMENTED** [review]
Looks good to merge

File: src/components/ArticlesList.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  render() {
+    const articlesIndex = this.state.articlesIndex
+    let showArticles
+
+    if (articlesIndex !== []) {
+      showArticles = articlesIndex.map(article => {
+        return (
+          <>
```
Comment (tochman) [inline src/components/ArticlesList.jsx]: No need to keep these fragments

File: src/components/ArticlesList.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            <div key={article.id} className='article'>
+              <div className=".article-title" id="title"><h3>{article.title}</h3></div>
+              <div className=".article-content" id="snippet"><p>{article.snippet}</p></div>
+            </div>
+          </>
+        )
+      })
+    }
+    return (
+      <div>
```
Comment (tochman) [inline src/components/ArticlesList.jsx]: No need to keep this div

Comment (Carltesio) [reply src/components/ArticlesList.jsx]: canceled them

File: src/components/ArticlesList.jsx (hunk 2 above)
Comment (Carltesio) [reply src/components/ArticlesList.jsx]: canceled

**Pull request merged**